apps/events-helsinki/browser-tests/roles/allCookiesUser.ts

```diff
@@ -0,0 +1,15 @@
+import { Role } from 'testcafe';
+import { getEnvUrl } from '../../../../packages/common-tests/browser-tests';
+import EventsConsentModal from '../page-models/events-consent-modal';
+
+export const acceptAllCookies = async () => {
+  const cookieConsentModal = new EventsConsentModal();
```

Comment:
Not very big thing but I checked why there is `EventsConsentModal` and `HobbiesConsentModal` separately and only difference is the title translation. It is checked only in `isOpened` method and now it's even commented out. So what I think is that if you would create the parent object `new ConsentModal();` instead then this file could be moved to shared code and not duplicated to each project. You have to remove the `abstract` from `ConsentModal` class though. Anyway I think it's okay and then we could have less copy-paste per project because for example on Liikunta you would have to do this role for the third time... 🧻 ✂️

Reply:
Hmm.. Let's see how they evolve. I add a todo-note in the code. Liikunta will have a different cookie set than the others.